hivemind/averaging/matchmaking.py

```diff
@@ -43,16 +46,16 @@ def __init__(
         prefix: str,
         target_group_size: int,
         min_group_size: int,
+        min_matchmaking_time: float,
         request_timeout: float,
         client_mode: bool,
         initial_group_bits: str = "",
-        averaging_expiration: float = 15,
```

Comment:
At some point, it'd be great to document this class as well, but you can do this in future PRs

Reply:
yep, i've been wanting to make something like the DHT structure scheme, but for averaging and optimizers